src/main/java/mate/academy/bookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -39,6 +39,13 @@ public ResponseEntity<Object> handleEntityNotFoundException(
         return new ResponseEntity<>(body, HttpStatus.NOT_FOUND);
     }
 
+    @ExceptionHandler(RegistrationException.class)
+    public ResponseEntity<Object> handleRegistrationException(
+            RegistrationException ex) {
+        Map<String, Object> body = generateErrorBody(ex, HttpStatus.UNAUTHORIZED);
+        return new ResponseEntity<>(body, HttpStatus.UNAUTHORIZED);
```

Comment:
It's a incorrect to return an `UNAUTHORIZED`  status code for `/registration` endpoint, because `UNAUTHORIZED` means that credentials that user passed to you are absent in database. Will be better to return `BAD_REQUEST` status with informative message